bin/node/executor/src/lib.rs

```diff
@@ -561,7 +561,7 @@ mod tests {
 				},
 				EventRecord {
 					phase: Phase::ApplyExtrinsic(2),
-					event: Event::pallet_treasury(pallet_treasury::RawEvent::Deposit(1984780231392)),
+					event: Event::pallet_treasury(pallet_treasury::RawEvent::Deposit(1984788199392)),
```

Comment:
I don't know exactly how this number is calculated, I just copied the output of the failing test...

I would assume that is reasonable to do in this scenario.

Reply:
maybe, but I wouldn't make that assumption, will have a look

Reply:
This comes from the `_4, Treasury,` part of the `pub type DealWithFees = SplitTwoWays` given to `transaction_payment::Trait`. Actually I think it is a great opportunity to dig into it and make it a useful and critical assertion: **it must be 20% of the total fee of a particular transaction**. 

I really think this is important to be asserted properly. If not fixed here, please make an issue (and assign me)